server/form-pages/apply/reasons-for-placement/basic-information/placementDate.test.ts

```diff
@@ -27,14 +25,14 @@ describe('PlacementDate', () => {
   })
 
   describe('constructor', () => {
-    describe('when the release date is in the future', () => {
+    describe('when the release date is after the application is created', () => {
```

Comment:
Suggestion: Do we need another test here to cover the case when the created at and released at are exactly the same? To cover this logic: `|| isSameDay(applicationDate, releaseDateObj)`